recipes/sbp/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+from textwrap import dedent
+
+import yaml
+
+from conans import ConanFile, CMake, tools
+
+
+class SbpConan(ConanFile):
+    name = "sbp"
+    license = "MIT"
+    author = "Swift Navigation <dev@swiftnav.com>"
```

Comment:
`author` is meant to be the author of the conan recipe.
But conan-center-index takes ownership of all recipes, so we don't add it.
```suggestion
```